paddle/fluid/operators/sequence_ops/sequence_enumerate_op.cc

```diff
@@ -62,6 +59,8 @@ class SequenceEnumerateOpMaker : public framework::OpProtoAndCheckerMaker {
         });
     AddAttr<int>("pad_value", "(int) The enumerate sequence padding value.")
         .SetDefault(0);
+    AddAttr<bool>(framework::kAllKernelsMustComputeRuntimeShape, "")
```

Comment:
doc?

Reply:
Since doc of `kAllKernelsMustComputeRuntimeShape` is so long and the same for all the ops, do you think we should put the doc in each op?

Currently, the doc is in `operator.h`: https://github.com/PaddlePaddle/Paddle/pull/16154/files#diff-f93a864675e978ec63f28873aa308a5cR71